File: fizzBuzz.go
package main// change this as needed

import (
	"fmt"
	"strconv"
)

func fizzBuzz(fizz int, buzz int, num int)  (string) {
	switch {
		case num % fizz == 0 && num % buzz == 0:
			return "FizzBuzz"
		case num % fizz == 0:
			return "Fizz"
		case num % buzz == 0:
			return "Buzz"
	}
	return strconv.Itoa(num)
}

func main() {
	for i := 0; i < 101; i++ {
		fmt.Println(fizzBuzz(3, 5, i))
	}
}                         
                              
  

File: fizzbuzz.py
import sys

def fizzBuzz(fizz, buzz, num):
	if num % fizz == 0 and num % buzz == 0:
		print("FizzBuzz")
	elif num % fizz == 0:
		print("Fizz")
	elif num % buzz == 0:
		print ("Buzz")
	else:
		print(num)

def main():
	for i in range(1,101):
		fizzBuzz(3, 5, i)


if __name__=="__main__":
	main()
